README.md

```diff
@@ -21,9 +20,13 @@ Install ODO extension manually
 5 Restart the node server inside PFE container to load the ODO extension
 Note: We already had installer for ODO extension, the installer will help you automatically install ODO extension if you are running Codewind on OKD/OpenShift cluster, this is only workaround when installer doesn't work properly
```

Comment:
Note: We already have an installer for the ODO extension. The installer will help you automatically install the ODO extension. If you are running Codewind on an OKD/OpenShift cluster, this is the only workaround when the installer does not work properly.